autoscheduler/scheduler/utils.py

```diff
@@ -55,6 +56,14 @@ def __eq__(self, other):
         return (self.start_time == other.start_time and self.end_time == other.end_time
                 and self.meeting_days == other.meeting_days)
 
+class BasicOptions(enum.Enum):
+    """ Represents the allowable values for the honors and web filters. These are
+        'include', 'exclude', and 'only'
+    """
+    INCLUDE = 'include'
```

Comment:
I think we should consider changing `include` to something like `no preference`, as I think it portrays that the option basically means give me whatever is possible / it indicates that you're going to get whatever section is possible better than `include` does, since `include`/`no preference` is basically the lack of a filter if that makes sense. Although at the end of the day we really just gotta see what the users think is intuitive

Reply:
Would you want that changed on the frontend as well?

Reply:
Yeah probably, just wanna stay consistent with our naming so it doesn't end up like the backend models that use `dept`/`subject` and `term`/`term_code` in different places